examples/lambda-layer/package.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "serverless-raw",
```

Comment:
Nit - but this name probably should change.

Reply:
done!